lib/pages/sign_up.dart

```diff
@@ -570,7 +575,22 @@ class _SignUpState extends State<SignUp> {
             ),
           ),
           SizedBox(
-            height: Dim.heightPercent(10),
+            height: Dim.heightPercent(3),
+          ),
+          GestureDetector(
+            child: Text(
+              AppLocalizations.of(context)!.contact_technical_support,
+              textAlign: TextAlign.center,
+              style: TextStyle(
+                fontSize: 17.0,
+                fontWeight: FontWeight.normal,
+                color: Color(0xFF004dff)
+              ),
+            ),
+            onTap: () => _launchHelpUrl(),
```

Comment:
helpUrl will be null, and nothing will happen, it will be confusing for the user, what do you think?

Reply:
Please help me the message for the feedback to user. It is not mention in the design so I dont reaction for that case

Reply:
I think we just can open the webview with crisp.chat support, we do this in settings screen "Customer support" button, but I will talk to @BastiaanVanGaalen in case of null